packages/mobile/src/components/countdown/helpers/interval-setter.ts

```diff
@@ -0,0 +1,26 @@
+import { getStrictTimeToEvent } from '~/helpers/date/date';
+
+type Time = {
+  days: string;
+  hours: string;
+  minutes: string;
+  seconds: string;
+};
+
+const intervalSetter = (
+  setState: (value: Time) => void,
+  endDate: Date,
+): number => {
+  setState(getStrictTimeToEvent(endDate));
+  const intervalId = setInterval(() => {
+    const { totalMs, ...duration } = getStrictTimeToEvent(endDate);
+    setState(duration);
+    if (totalMs < 1) {
+      clearInterval(intervalId);
+    }
+  }, 1000);
```

Comment:
Such numbers are called magic cause it is not clear what exactly they mean. So you should create a constant instead. F.e.
```
const UPDATE_END_DATE_INTERVAL = 1000
```
and use it there instead of just the number 1000. The same for 1 in totalMs comparison. Make constant and name it properly, so it is clear what exactly it is

Reply:
Fixed. Thank you for the explanation .